styles.css

```diff
@@ -21,12 +38,16 @@ body {
   text-decoration: none;
 }
 
+.menu-list-items > ul {
+  display: none;
+}
+
 .intro {
```

Comment:
- [ ] As the previous code reviewer mentioned, kindly make the content of your first section wider as the Figma template so it does not leave a lot of whitespace around the texts.
 - Your design:
 - 
<img width="1359" alt="image" src="https://user-images.githubusercontent.com/61919617/230084037-171bb046-4afc-4275-8ec0-9821e7b678f2.png">

 - Figma tempalte:
 - 
<img width="630" alt="image" src="https://user-images.githubusercontent.com/61919617/230084109-c5ebd3ac-d08a-4e39-a57c-d9bbdce0856d.png">

Reply:
Could you check one more time this point